src/api/v1/router.ts

```diff
@@ -21,6 +22,20 @@ import { ERROR_CODES } from '../../constants/errors';
 const router: Router = Router();
 export default router;
 
+const jwtMiddlewareOptions: JWTMiddlewareOptions = {
+  jwtOptions: {
+    jwksUrl: `${process.env.AUTH_API_URL}/.well-known/jwks.json`,
+    issuer: process.env.AUTH_ISSUER || ''
```

Comment:
I need these variables to be added to 1password so I can configure them for the Enclave in the Evervault dashboard, before we can deploy.

Reply:
Those environment variables have been added to 1Password.
@K-Mistele 
I noticed that AWS Access Key and AWS Secret Key are already being used in the enclave environment. It might be better to integrate AWS Secrets Manager into the nitro-secure service and create the appropriate secrets there. Does that make sense?